docs/src/icons/SvgDiscord.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import RootSvg, { RootSvgProps } from 'docs/src/icons/RootSvg';
+
+function SvgComponent(props: RootSvgProps) {
+  return (
+    <RootSvg
+      viewBox="0 0 24 24"
+      xmlns="http://www.w3.org/2000/svg"
+      fill="currentColor"
+      width="20"
+      height="20"
```

Comment:
I think to consider what's the best-balanced area for each icon, relative to https://mui.com/material-ui/material-icons/?query=brand. Maybe this would make sense:

```suggestion
      width="19"
      height="19"
```